examples/models/population_server.py

```diff
@@ -48,20 +49,24 @@ def pyramid():
     female_quad = Quad(left=0, right="female", bottom="groups", top="shifted", fill_color="#CFF09E")
     female_quad_glyph = plot.add_glyph(source_pyramid, female_quad)
 
-    plot.add_layout(Legend(legends=[("Male", [male_quad_glyph]), ("Female", [female_quad_glyph])]))
+    plot.add_layout(Legend(items=[
+        LegendItem(label="Male", renderers=[male_quad_glyph]),
+        LegendItem(label="Female", renderers=[female_quad_glyph])
+    ]))
```

Comment:
I think there's a few variations on this which could make it quicker to spell / might be people's guess:

List on glyph_renderer optional:  
```python
items = [("Male", male_quad_glyph), ("Female", female_quad_glyph)]
```

Dictionary (a few ways)

```python
items = {"Male": male_quad_glyph, "Female", female_quad_glyph}
```

```python
items = {"Male": [male_quad_glyph], "Female", [female_quad_glyph]}
```

```python
items = [{"Male": [male_quad_glyph]}, {"Female", [female_quad_glyph]}]
```